src/helpers/byDisplayValue.ts

```diff
@@ -7,9 +7,9 @@ import { createLibraryNotSupportedError } from './errors';
 import type { TextMatchOptions } from './byText';
 
 const getTextInputNodeByDisplayValue = (
-  node,
-  value,
-  options?: TextMatchOptions = {}
+  node: ReactTestInstance,
+  value: string | RegExp,
```

Comment:
I think this particular union may deserve an alias because of its widespread use. WDYT @thymikee?